bedrock/firefox/templates/firefox/includes/download-button.html

```diff
@@ -41,73 +41,41 @@
-          <span class="download-content">
-            {% if plat.os == 'android' %}
-              {% if channel == 'beta' %}
-                <strong class="download-title">{{ _('<span>Firefox Beta</span> for Android') }}</strong>
-              {% elif channel == 'alpha' %}
-                <strong class="download-title">{{ _('<span>Firefox Aurora</span> for Android') }} {{ plat.arch_pretty }}</strong>
-              {% else %}
-                <strong class="download-title">{{ _('<span>Firefox</span> for Android') }}</strong>
-              {% endif %}
-              {% if channel != 'alpha' %}
-                <span class="download-subtitle">{{ _('Get it free on Google Play') }}</span>
-              {% endif %}
-            {% elif plat.os == 'ios' %}
-              <strong class="download-title">{{ _('<span>Firefox</span> for iOS') }}</strong>
-              <span class="download-subtitle">{{ _('Get it free from the App Store') }}</span>
-            {% else %}
-              {% if channel == 'beta' %}
-                <strong class="download-title">{{ _('Firefox Beta') }}</strong>
-              {% elif channel == 'alpha' %}
-                <strong class="download-title">{{ _('<span>Firefox</span> Developer Edition') }}</strong>
-              {% else %}
-                <strong class="download-title">{{ _('Firefox') }}</strong>
-              {% endif %}
-              <span class="download-subtitle">{{ _('Free Download') }}</span>
-              <span class="download-lang">{{ locale_name|safe }}</span>
-              <span class="download-platform">{{ plat.os_pretty }}</span>
-            {% endif %}
-          </span>
+
+           {% if alt_copy %}
+             <strong class="download-title">{{ alt_copy }}</strong>
+           {% else %}
+             {% if plat.os == 'android' %}
+               {% if channel == 'beta' %}
+                 <strong class="download-title">{{ _('<span>Firefox Beta</span> for Android') }}</strong>
+               {% elif channel == 'alpha' %}
+                 <strong class="download-title">{{ _('<span>Firefox Aurora</span> for Android') }}</strong>
+               {% else %}
+                 <strong class="download-title">{{ _('<span>Firefox</span> for Android') }}</strong>
+               {% endif %}
+             {% elif plat.os == 'ios' %}
+               <strong class="download-title">{{ _('<span>Firefox</span> for iOS') }}</strong>
+             {% else %}
+               {% if channel == 'beta' %}
+                 <strong class="download-title">{{ _('Firefox Beta') }}</strong>
+               {% elif channel == 'alpha' %}
+                 <strong class="download-title">{{ _('<span>Firefox</span> Developer Edition') }}</strong>
+               {% else %}
+                 <strong class="download-title">{{ _('Download Firefox') }}</strong>
+               {% endif %}
+             {% endif %}
+          {% endif %}
```

Comment:
Unless white space would cause an issue, would be nice to not repeat the `<strong class="download-title">` tag for each variation. Maybe try wrapping the logic in the `<strong>` tag instead?